codespan-reporting/src/emitter.rs

```diff
@@ -1,23 +1,92 @@
+use std::io;
+
 use codespan::CodeMap;
 
+use termcolor::{Color, ColorSpec, WriteColor};
+
 use Diagnostic;
 
-pub fn emit(codemap: &CodeMap, diagnostic: &Diagnostic) {
-    println!("{}: {}", diagnostic.severity, diagnostic.message);
+pub fn emit<W>(mut writer: W, codemap: &CodeMap, diagnostic: &Diagnostic) -> io::Result<()>
```

Comment:
Goood. Was wondering if I should switch it to be `eprintln!`, but this is better!

Reply:
So is `WriteColor` configurable? ie. my repl has a `--color=[auto|always|never]` flag - would I be able to set that?

Reply:
Yup, you just set `ColorChoice` when you create the `WriteColor` instance https://github.com/brendanzab/codespan/pull/5/files#diff-a45cab6253488a1ff30335b30b8dc71aR38

Reply:
niiiice